dagger/conf.nim

```diff
@@ -66,17 +68,38 @@ type
       defaultValue: noCommand }: StartUpCommand
 
     of noCommand:
-      listenAddrs* {.
-        desc: "Specifies one or more listening multiaddrs for the node to listen on."
-        defaultValue: @[MultiAddress.init("/ip4/0.0.0.0/tcp/0").tryGet()]
-        defaultValueDesc: "/ip4/0.0.0.0/tcp/0"
-        abbr: "a"
-        name: "listen-addrs" }: seq[MultiAddress]
+      listenPorts* {.
+        desc: "Specifies one or more listening ports for the node to listen on."
+        defaultValue: @[Port(0)]
+        defaultValueDesc: "0"
+        abbr: "l"
+        name: "listen-ports" }: seq[Port]
```

Comment:
Why do we need more than one port to listen on?

Reply:
Because we can :) Before this PR, multiples MAs were accepted, now I accept multiple port, not sure if it's interesting to anyone, but it's cheap to do so..